docs/guides/1.18.0-upgrade-guide.md

```diff
@@ -4,7 +4,7 @@ page_title: "Upgrade Guide 1.18.0"
 
 # MongoDB Atlas Provider 1.18.0: Upgrade and Information Guide
 
-***WARNING:*** For users using the `mongodbatlas_advanced_cluster` resource or data sources, and defining multi sharded clusters, this new version impacts the possibility of editing the definition of these clusters from the Atlas UI. This impact is limited to the first weeks of August 2024.
+***WARNING:*** For users using the `mongodbatlas_advanced_cluster` resource or data sources, and defining multi sharded clusters, this new version impacts the possibility of editing the definition of these clusters from the Atlas UI. This impact is limited to the first weeks of September 2024.
```

Comment:
can you update the changelog entry that also points this out as well? As it is part of a version that is already released you can update the CHANGELOG.md file directly.

Reply:
updated